### PR TITLE
VZ-11376: Build NGINX images using Golang v1.20.10

### PR DESCRIPTION
**Pull request by bhatpmk**
Build ingress controller images using Go 1.20.10

### CODE REVIEW

**Review by kminder — APPROVED** [review]
Approved assuming you have confirmed existing PLS approval for all go.mod changes to direct dependencies.

**Comment by bhatpmk** [comment]
> Approved assuming you have confirmed existing PLS approval for all go.mod changes to direct dependencies.

I have submitted this change, will follow-up to understand the dependencies are good with the existing PLS approval.

**Pull request merged**
